Makefile

```diff
@@ -54,12 +58,23 @@ install: manifests
 uninstall: manifests
 	kustomize build config/crd | kubectl delete -f -
 
-# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy:
-	pushd config/manager
+# TODO(wallrj): .PHONY ensures that the install file is always regenerated,
+# because I this really depends on the checksum of the Docker image and all the
```

Comment:
Does this mean you want to ensure the docker image doesn't contain anything stale? I can't make sense of this sentence

Reply:
I meant that without `.PHONY` the `build/install.yaml` file will be built once and never regenerated when either the kustomize `config/` files change *OR* when the Docker image version number changes.

I could create a Make rule for all those files and all the `.go` files, using `find`, I suppose.
But this seemed easier, for now.